Exercise_D.js

```diff
@@ -14,7 +14,18 @@
 Use a combination of conditionals and loops to create your answer for this question. */
 
 function multiplyNumber(number) {
-  // CODE HERE
+  if(number<=0 || typeof("number")!="number"){
+    return "ERROR"
+  }else if (number>=1000000){
+    return number;
+  }else if (number<1000000 && number>0){
+     for (let x= number;x<1000000 ;x*10){
+       if(i>=1000000){
+         return x;
+       }
+     }
+
+  }
 }
 
 console.log(multiplyNumber(10));

```

Comment:
For this exercise, it did not run properly. As it gave an ERROR with the parameter of 10. 
Let's break this down, for the first part you would need to check if the number is smaller than 0 or not a number 
You can use `isNan( ) and also number <= 0`as the first condition. For the second condition, you almost got it correct. But the condition should be `number > 1000000`. The ending condition is to check, if the number is smaller than 1000000, and have it multiply by 10 until it reaches that number. 
You can use `number < 1000000` in a while loop, to keep on running until it reaches that number.

Reply:
Also, i would like to add that Exercise E & F were not completed